patch out unreachable code that yields linking problems (CLIM_IMPORT and CLIM_EXPORT are not available)
CPHAN is hard coded to 'SIPC' in arp/module/yom_oas.F90
this code is optimized out by gfortran, but not by ifort
--- ../arp/climate/updcpl.F90.orig	2012-12-29 09:58:32.004452610 +0100
+++ ../arp/climate/updcpl.F90	2012-12-29 09:58:53.836681331 +0100
@@ -311,16 +311,16 @@
 
         CALL SIPC_READ_MODEL(JV,NGPTOTG,CLJOBNAM_R,INFOS,ZFIELDBUF)
 
-      ELSEIF( CPHAN == 'MPI2') THEN
-
-        CALL CLIM_IMPORT(CL_READ(JV),ISTEPCPL,ZFIELDBUF,INFO)
-        IF(INFO /= Z_CLIM_OK) then
-          WRITE(NULOUT,*) 'PB in READING ',CL_READ(JV),JV
-          WRITE(NULOUT,*) 'COUPLAGE STEP IS =',ISTEPCPL
-          WRITE(NULOUT,*) 'CLIM ERROR CODE IS =',INFO
-          CALL FLUSH(NULOUT)
-          CALL ABOR1('UPDCPL: ABOR1 CALLED CLIM_IMPORT')
-        ENDIF
+!      ELSEIF( CPHAN == 'MPI2') THEN
+!
+!        CALL CLIM_IMPORT(CL_READ(JV),ISTEPCPL,ZFIELDBUF,INFO)
+!        IF(INFO /= Z_CLIM_OK) then
+!          WRITE(NULOUT,*) 'PB in READING ',CL_READ(JV),JV
+!          WRITE(NULOUT,*) 'COUPLAGE STEP IS =',ISTEPCPL
+!          WRITE(NULOUT,*) 'CLIM ERROR CODE IS =',INFO
+!          CALL FLUSH(NULOUT)
+!          CALL ABOR1('UPDCPL: ABOR1 CALLED CLIM_IMPORT')
+!        ENDIF
       ENDIF
 
       CALL DISGRID_SEND(1,ZFIELDBUF,INUM,ZFIELD(:,JV))
--- ../arp/ocean/wrcpl.F90.orig	2012-12-29 10:02:50.736177120 +0100
+++ ../arp/ocean/wrcpl.F90	2012-12-29 10:02:53.733208569 +0100
@@ -183,9 +183,9 @@
       IF (CPHAN == 'SIPC') THEN
         CALL SIPC_WRITE_MODEL(INDEX1,IDIMG,INFOS,ZREALG)
         CALL SIPC_WRITE_MODEL(INDEX2,IDIMG,INFOS,ZREALG)
-      ELSEIF (CPHAN == 'MPI2') THEN
-        CALL CLIM_EXPORT(CL_WRIT(INDEX1),ISTEP,ZREALG,INFO)
-        CALL CLIM_EXPORT(CL_WRIT(INDEX2),ISTEP,ZREALG,INFO)
+!      ELSEIF (CPHAN == 'MPI2') THEN
+!        CALL CLIM_EXPORT(CL_WRIT(INDEX1),ISTEP,ZREALG,INFO)
+!        CALL CLIM_EXPORT(CL_WRIT(INDEX2),ISTEP,ZREALG,INFO)
       ENDIF
     ENDIF
   ENDIF
@@ -218,9 +218,9 @@
       IF (CPHAN == 'SIPC') THEN
         CALL SIPC_WRITE_MODEL(INDEX1,IDIMG,INFOS,ZREALG)
         CALL SIPC_WRITE_MODEL(INDEX2,IDIMG,INFOS,ZREALG)
-      ELSEIF (CPHAN == 'MPI2') THEN
-        CALL CLIM_EXPORT(CL_WRIT(INDEX1),ISTEP,ZREALG,INFO)
-        CALL CLIM_EXPORT(CL_WRIT(INDEX2),ISTEP,ZREALG,INFO)
+!      ELSEIF (CPHAN == 'MPI2') THEN
+!        CALL CLIM_EXPORT(CL_WRIT(INDEX1),ISTEP,ZREALG,INFO)
+!        CALL CLIM_EXPORT(CL_WRIT(INDEX2),ISTEP,ZREALG,INFO)
       ENDIF
     ENDIF
   ENDIF
@@ -248,8 +248,8 @@
       INDEX=1
       IF (CPHAN == 'SIPC') THEN
         CALL SIPC_WRITE_MODEL(INDEX,IDIMG,INFOS,ZREALG)
-      ELSEIF (CPHAN == 'MPI2') THEN
-        CALL CLIM_EXPORT(CL_WRIT(INDEX),ISTEP,ZREALG,INFO)
+!      ELSEIF (CPHAN == 'MPI2') THEN
+!        CALL CLIM_EXPORT(CL_WRIT(INDEX),ISTEP,ZREALG,INFO)
       ENDIF
     ENDIF
   ENDIF
@@ -277,8 +277,8 @@
       INDEX=2
       IF (CPHAN == 'SIPC') THEN
         CALL SIPC_WRITE_MODEL(INDEX,IDIMG,INFOS,ZREALG)
-      ELSEIF (CPHAN == 'MPI2') THEN
-        CALL CLIM_EXPORT(CL_WRIT(INDEX),ISTEP,ZREALG,INFO)
+!      ELSEIF (CPHAN == 'MPI2') THEN
+!        CALL CLIM_EXPORT(CL_WRIT(INDEX),ISTEP,ZREALG,INFO)
       ENDIF
     ENDIF
   ENDIF
@@ -312,8 +312,8 @@
       INDEX=3
       IF (CPHAN == 'SIPC') THEN
         CALL SIPC_WRITE_MODEL(INDEX,IDIMG,INFOS,ZREALG)
-      ELSEIF (CPHAN == 'MPI2') THEN
-        CALL CLIM_EXPORT(CL_WRIT(INDEX),ISTEP,ZREALG,INFO)
+!      ELSEIF (CPHAN == 'MPI2') THEN
+!        CALL CLIM_EXPORT(CL_WRIT(INDEX),ISTEP,ZREALG,INFO)
       ENDIF
     ENDIF
   ENDIF
@@ -347,8 +347,8 @@
       INDEX=4
       IF (CPHAN == 'SIPC') THEN
         CALL SIPC_WRITE_MODEL(INDEX,IDIMG,INFOS,ZREALG)
-      ELSEIF (CPHAN == 'MPI2') THEN
-        CALL CLIM_EXPORT(CL_WRIT(INDEX),ISTEP,ZREALG,INFO)
+!      ELSEIF (CPHAN == 'MPI2') THEN
+!        CALL CLIM_EXPORT(CL_WRIT(INDEX),ISTEP,ZREALG,INFO)
       ENDIF
     ENDIF
   ENDIF
@@ -376,8 +376,8 @@
       INDEX=9
       IF (CPHAN == 'SIPC') THEN
         CALL SIPC_WRITE_MODEL(INDEX,IDIMG,INFOS,ZREALG)
-      ELSEIF (CPHAN == 'MPI2') THEN
-        CALL CLIM_EXPORT(CL_WRIT(INDEX),ISTEP,ZREALG,INFO)
+!      ELSEIF (CPHAN == 'MPI2') THEN
+!        CALL CLIM_EXPORT(CL_WRIT(INDEX),ISTEP,ZREALG,INFO)
       ENDIF
     ENDIF
   ENDIF
@@ -413,8 +413,8 @@
     ELSE
       IF (CPHAN == 'SIPC') THEN
         CALL SIPC_WRITE_MODEL(INDEX,IDIMG,INFOS,ZREALG)
-      ELSEIF (CPHAN == 'MPI2') THEN
-        CALL CLIM_EXPORT(CL_WRIT(INDEX),ISTEP,ZREALG,INFO)
+!      ELSEIF (CPHAN == 'MPI2') THEN
+!        CALL CLIM_EXPORT(CL_WRIT(INDEX),ISTEP,ZREALG,INFO)
       ENDIF
     ENDIF
   ENDIF
@@ -446,8 +446,8 @@
     ELSE
       IF (CPHAN == 'SIPC') THEN
         CALL SIPC_WRITE_MODEL(INDEX,IDIMG,INFOS,ZREALG)
-      ELSEIF (CPHAN == 'MPI2') THEN
-        CALL CLIM_EXPORT(CL_WRIT(INDEX),ISTEP,ZREALG,INFO)
+!      ELSEIF (CPHAN == 'MPI2') THEN
+!        CALL CLIM_EXPORT(CL_WRIT(INDEX),ISTEP,ZREALG,INFO)
       ENDIF
     ENDIF
   ENDIF
@@ -487,10 +487,10 @@
         CALL SIPC_WRITE_MODEL(INDEX,IDIMG,INFOS,ZREALG)
         CALL SIPC_WRITE_MODEL(INDEX1,IDIMG,INFOS,ZREALG)
         CALL SIPC_WRITE_MODEL(INDEX2,IDIMG,INFOS,ZREALG)
-      ELSEIF (CPHAN == 'MPI2') THEN
-        CALL CLIM_EXPORT(CL_WRIT(INDEX),ISTEP,ZREALG,INFO)
-        CALL CLIM_EXPORT(CL_WRIT(INDEX1),ISTEP,ZREALG,INFO)
-        CALL CLIM_EXPORT(CL_WRIT(INDEX2),ISTEP,ZREALG,INFO)
+!      ELSEIF (CPHAN == 'MPI2') THEN
+!        CALL CLIM_EXPORT(CL_WRIT(INDEX),ISTEP,ZREALG,INFO)
+!        CALL CLIM_EXPORT(CL_WRIT(INDEX1),ISTEP,ZREALG,INFO)
+!        CALL CLIM_EXPORT(CL_WRIT(INDEX2),ISTEP,ZREALG,INFO)
       ENDIF
     ENDIF
   ENDIF
@@ -522,8 +522,8 @@
     ELSE
       IF (CPHAN == 'SIPC') THEN
         CALL SIPC_WRITE_MODEL(INDEX,IDIMG,INFOS,ZREALG)
-      ELSEIF (CPHAN == 'MPI2') THEN
-        CALL CLIM_EXPORT(CL_WRIT(INDEX),ISTEP,ZREALG,INFO)
+!      ELSEIF (CPHAN == 'MPI2') THEN
+!        CALL CLIM_EXPORT(CL_WRIT(INDEX),ISTEP,ZREALG,INFO)
       ENDIF
     ENDIF
   ENDIF
@@ -550,8 +550,8 @@
     ELSE
       IF (CPHAN == 'SIPC') THEN
         CALL SIPC_WRITE_MODEL(INDEX,IDIMG,INFOS,ZREALG)
-      ELSEIF (CPHAN == 'MPI2') THEN
-        CALL CLIM_EXPORT(CL_WRIT(INDEX),ISTEP,ZREALG,INFO)
+!      ELSEIF (CPHAN == 'MPI2') THEN
+!        CALL CLIM_EXPORT(CL_WRIT(INDEX),ISTEP,ZREALG,INFO)
       ENDIF
     ENDIF
   ENDIF
